cpp/src/bindings.cpp

```diff
@@ -640,4 +640,43 @@ PYBIND11_MODULE(_pypowsybl, m) {
     m.def("remove_aliases", &pypowsybl::removeAliases, "remove specified aliases on a network", py::arg("network"), py::arg("dataframe"));
 
     m.def("close", &pypowsybl::closePypowsybl, "Closes pypowsybl module.");
+    /************************DYNAMIC SIMULATION WITH DYNAWALTZ*******************************/
+    //TODO descriptions of methods
+
+    //entrypoints for constructors
+    m.def("create_dynamic_simulation_context", &pypowsybl::createDynamicSimulationContext, "TODO desc");
+    m.def("create_dynamic_model_mapping", &pypowsybl::createDynamicModelMapping, "TODO desc");
+    m.def("create_timeseries_mapping", &pypowsybl::createTimeseriesMapping, "TODO desc");
+    m.def("create_event_mapping", &pypowsybl::createEventMapping, "TODO desc");
+
+    //running simulations
+    m.def("run_dynamic_model", &pypowsybl::runDynamicModel, "Run a dynamic model with Dynawaltz", py::call_guard<py::gil_scoped_release>(),
+        py::arg("dynamic_model"), py::arg("network"), py::arg("dynamic_mapping"), py::arg("event_mapping"), py::arg("timeseries_mapping"), py::arg("start"), py::arg("stop"));
+
+    //model mapping
+    m.def("add_alphabeta_load", &pypowsybl::addAlphaBetaLoad, "TODO desc", py::arg("dynamic_mapping_handle"), py::arg("static_id"), py::arg("dynamic_param"));
```

Comment:
I think instead of having multiple methods here, we could have only one for all models, which would receive a dataframe input exactly as we have done for all updates methods of the network. 
This will allow to:
- minimize the code boiler plate necessary for each kind of event
- map multiple models in only one call. This is very important for performances, since calling 100s of methods in a for loop will have very poor performance in python.

Reply:
(but of course we still need to have multiple methods on the python side, for the user api

Reply:
The change has been done here : f90ea09cb29b308e0b84299496bc1948c0bb7bff .
However some mappings like CurrentLimitAutomaton (and maybe other mapping in the future) need more infos.

The actual behavior is to have some default values (possible because CurrentLimitAutomaton needs an enum argument) to be able to map them _via_ the dataframe call.
But they still need a special python/java binding to be able to specify all the arguments.